content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1066,28 +1077,27 @@ the list type:
-| 'boolean'                                          | boolean                                                                                                                      |
-| 'number' (all formats)                             | double                                                                                                                       |
-| 'integer' (all formats)                            | int (64)                                                                                                                     |
-| 'null'                                             | null_type                                                                                                                    |
-| 'string'                                           | string                                                                                                                       |
-| 'string' with format=byte (base64 encoded)         | bytes                                                                                                                        |
-| 'string' with format=date                          | timestamp (google.protobuf.Timestamp)                                                                                        |
-| 'string' with format=datetime                      | timestamp (google.protobuf.Timestamp)                                                                                        |
-| 'string' with format=duration                      | duration (google.protobuf.Duration)                                                                                          |
+| OpenAPIv3 type | CEL type |
+| -------------- | -------- |
+| 'object' with Properties | object / "message type" |
+| 'object' with AdditionalProperties | map |
+| 'object' with x-kubernetes-embedded-type | object / "message type", 'apiVersion', 'kind', 'metadata.name' and 'metadata.generateName' are implicitly included in schema |
+| 'object' with x-kubernetes-preserve-unknown-fields | object / "message type", unknown fields are NOT accessible in CEL expression |
+| x-kubernetes-int-or-string | dynamic object that is either an int or a string, `type(value)` can be used to check the type |
+| 'array | list |
+| 'array' with x-kubernetes-list-type=map | list with map based Equality & unique key guarantees |
+| 'array' with x-kubernetes-list-type=set | list with set based Equality & unique entry guarantees |
+| 'boolean' | boolean |
+| 'number' (all formats) | double |
+| 'integer' (all formats) | int (64) |
+| 'null' | null_type |
+| 'string' | string |
+| 'string' with format=byte (base64 encoded) | bytes |
+| 'string' with format=date | timestamp (google.protobuf.Timestamp) |
+| 'string' with format=datetime | timestamp (google.protobuf.Timestamp) |
+| 'string' with format=duration | duration (google.protobuf.Duration) |
```

Comment:
For tables with cells too wide, I'd suggest we switch back to HTML tables.

Reply:
It's a html table now.